### PR TITLE
Push up to date

### PR DESCRIPTION
**Pull request by jgrund**
Add all commits up to master

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,12 +1,9 @@
-node_modules
```
Comment (johnsonw) [inline .npmignore]: Why was node_modules removed?

File: .eslintignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 dist
 node_modules
 coverage
+bin/test.js
```
Comment (johnsonw) [inline .eslintignore]: This doesn't seem right. Are we using bin/test.js? It looks like it was removed below.

File: .npmignore (hunk 1 above)
Comment (jgrund) [inline .npmignore]: Because it's not needed.

File: .eslintignore (hunk 2 above)
Comment (jgrund) [inline .eslintignore]: Fixing.

**Pull request merged**